src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -65,8 +65,9 @@ AsyncRPCOperation_sendmany::AsyncRPCOperation_sendmany(
         std::vector<SendManyRecipient> zOutputs,
         int minDepth,
         CAmount fee,
-        UniValue contextInfo) :
-        tx_(contextualTx), fromaddress_(fromAddress), t_outputs_(tOutputs), z_outputs_(zOutputs), mindepth_(minDepth), fee_(fee), contextinfo_(contextInfo)
+        UniValue contextInfo,
+        bool returnChangeToSenderAddr) :

```

Comment:
Just FYI: C++ now allows to use in constructor variables with same names as class members.
Also, can use "const bool bReturnChangeToSenderAddr" here.

Reply:
fixed as suggestion.